src/sentry/search/utils.py

```diff
@@ -18,6 +18,15 @@ class InvalidQuery(Exception):
     pass
 
 
+def parse_release(project, value):
+    # TODO(dcramer): add environment support
+    if value == 'latest':
```

Comment:
I feel like we should make sure there isn't a Release actually named `latest` first before assuming it's the latest by time.